x-pack/plugins/fleet/dev_docs/developing_kibana_in_serverless.md

```diff
@@ -0,0 +1,89 @@
+# Developing Kibana in serverless mode
+
+Fleet is enabled for the observability and security serverless project types.
+
+To run Elasticsearch and Kibana in serverless mode, the relevant commands are:
+
+For the observability project type:
+```bash
+# Start Elasticsearch in serverless mode as an observability project
+yarn es serverless --projectType=oblt --kill
+
+# Run Kibana as an observability project
+yarn serverless-oblt
+```
+
+and one of:
+
+```bash
+# Start Elasticsearch in serverless mode as a security project
+yarn es serverless --projectType=security --kill
+
+# Run Kibana as a security project
+yarn serverless-security
+```
+
+Once running, you can login with the username `elastic_serverless` or `system_indices_superuser` and the password `changeme`.
```

Comment:
I have in my notes to launch kibana on this url: `https://0.0.0.0:5601/`, I think it doesn't work on the same url as the stateful version `http://localhost:5601/xyz`

Reply:
Ah, good point. I have been successfully `http://localhost:5601/` but I think the base path doesn't work. I can add a comment on that.

Reply:
Pushed - also, it looks like I forgot to hit save earlier, to the bit about running a fleet server and enrolling agents is now there too 🙈

Reply:
actually `https` is only needed if kibana is started with the `--ssl` flag, which is optional